test/test_shared_ptr.vcxproj

```diff
@@ -0,0 +1,8 @@
+<Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
I think "ToolsVersion" is lower because we're still stuck on 2013. ? Maybe 12.0

Reply:
I took it from here but I think you're right: https://github.com/sass/libsass/blob/9740de67d01d1f9da0278dca1ffad984304f9124/win/libsass.vcxproj#2

This [MSDN article](https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-toolset-toolsversion?view=vs-2019) says:

> Visual Studio 2015 uses ToolsVersion 14.0